content/app/development/logic/expressions/_index.nb.md

```diff
@@ -0,0 +1,266 @@
+---
+title: Dynamiske uttrykk
+linktitle: Uttrykk
+description: Oversikt over dynamiske uttrykk definert i JSON
+tags: [dynamics]
+toc: true
+---
+
+{{% panel theme="warning" %}}
+⚠️ Dynamikk er et område under aktiv utvikling. Denne funksjonaliteten er ikke tilgjengelig for konfigurasjon direkte
+i Altinn Studio enda, og må derfor skrives manuelt i JSON-filene.
+{{% /panel %}}
+
+## Introduksjon
+
+Dynamikk via uttrykk gjør det mulig å definere enkel dynamisk oppførsel i en Altinn 3 app, som for eksempel ved å
+definere om et skjemafelt skal vises eller skjules, om feltet skal være påkrevd eller skrivebeskyttet.
+
+Uttrykkene er tilgjengelige i alle Altinn 3-apper som bruker frontend-versjon
+[3.54.0](https://github.com/Altinn/app-frontend-react/releases/tag/v3.54.0) eller nyere. Bruker man denne versjonen
+(eller siste hovedversjon) har man mulighet til å benytte dynamiske uttrykk til [flere bruksområder](#bruksområder).
+
+Fra versjon `7.2.0` av [nuget-pakkene](../../../maintainance/dependencies#nuget) er også uttrykkene støttet i backend.
+Det gjør at serveren vil kunne evaluere uttrykkene og fjerne data som potensielt er lagret i instansen og er knyttet
+til felter/komponenter som i ettertid er skjult. Disse dataene fjernes fra datamodellen når instansen sendes inn.
+Det gjør det også mulig å unnlate å sende inn data som ellers er tilknyttet påkrevde felter - dersom disse påkrevde
+feltene er skjult i skjemaet ved hjelp av dynamiske uttrykk. Dette gjelder også ved innsending direkte fra API.
```

Comment:
I første omgang er det opt inn for å kjøre uttrykkene backend. Du må sette `"RemoveHiddenDataPreview": true` i `"AppSettings"` seksjonen i `appsettings.json`. Fikk litt panikk for at alt skulle brekke når det skulle i prod, og det var veldig enkelt å legge til en feature toggle

![image](https://user-images.githubusercontent.com/131616/202637818-7207680e-60fd-4027-886f-377e42b2a1aa.png)